js/qz-tray.js

```diff
@@ -1101,7 +1101,7 @@ var qz = (function() {
              *  @param {number} [options.copies=1] Number of copies to be printed.
              *  @param {number|Array<number>} [options.density=0] Pixel density (DPI, DPMM, or DPCM depending on <code>[options.units]</code>).
              *      If provided as an array, uses the first supported density found (or the first entry if none found).
-             *  @param {boolean} [options.duplex=false] Double sided printing
+             *  @param {boolean|string} [options.duplex=false] Double sided printing, Can specify duplex style by passing a string value: <code>[ONE-SIDED | DUPLEX | LONG-EDGE | TUMBLE | SHORT-EDGE]</code>
```

Comment:
Would it be more consistent to use `long-edge` (lowercase) to match the likes of `nearest-neighbor`, etc?
https://github.com/qzind/tray/blob/1a3e93f11f1bb86382b8b657bf4caac0281e3321/js/qz-tray.js#L1106